cognite/client/data_classes/simulators/models.py

```diff
@@ -354,6 +354,21 @@ def _get_update_properties(cls, item: CogniteResource | None = None) -> list[Pro
         ]
 
 
+@dataclass
+class SimulatorExternalDependencyFileReference(CogniteObject): ...
+
+
+@dataclass
+class SimulatorExternalDependencyFileInternalId(SimulatorExternalDependencyFileReference):
```

Comment:
Maybe just FileInternalId?
```suggestion
class FileInternalId(SimulatorExternalDependencyFileReference):
```